system/jlib/jtrace.cpp

```diff
@@ -789,11 +1031,13 @@ class CTraceManager : implements ITraceManager, public CInterface
                 processor = opentelemetry::sdk::trace::BatchSpanProcessorFactory::Create(std::move(exporter), options);
                 DBGLOG("OpenTel tracing using batch Span Processor");
             }
+            else if (foundProcessorType &&  strcmp("simple", processorType.str())==0)
+            {
+                DBGLOG("OpenTel tracing using batch simple Processor");
```

Comment:
These log statements will show up if the default trace config is overwritten in any way. @afishbeck noticed these log statements show up in clients which should not generate any output to std/str.
It turns out tracemanager init occurs before components have a chance to redirect/suppress jlog output... outside of the scope of this issue, but needs to be discussed.